src/config.h

```diff
@@ -143,7 +130,7 @@
 #define ESOCKTNOSUPPORT 0
 #endif
 
-#if ((defined __linux && defined(__GLIBC__)) || defined __APPLE__)
```

Comment:
do you have a clue why `__GLIBC__` was used here?
btw, what's the difference between `__linux` and `__linux__`?

Reply:
Tracing back, `__GLIBC__` was a mistake IMHO. I don't think there's significance about `__linux`, the more commonly used define is really `__linux__`.